### PR TITLE
Update settingsKey descriptions and links

### PR DESCRIPTION
**Pull request by blackbaud-johnly**
Addresses https://github.com/blackbaud/skyux2-docs/issues/514.

### CODE REVIEW

File: src/app/components/grid/index.html [hunk 1]
```diff
@@ -80,7 +80,7 @@
...
+    Specifies a unique key for the UI Config Service to retrieve stored settings from a database. You must prefix the key with the name of your SPA. The UI Config Service saves configuration settings for users and returns <sky-code>selectedColumnIds</sky-code> to preserve their preferred column order and the sorting of columns. This property accepts <sky-code>string</sky-code> values. This property is for internal Blackbaud use only. For Blackbaud developers, the UI Config Service website describes how to <a href="https://docs.blackbaud.com/ui-config-service-docs/learn/integrations/skyux-components/grids">apply sticky settings to grids</a>.
```
Comment (Blackbaud-AlexKingman) [inline src/app/components/grid/index.html]: > You must prefix the key with the name of your SPA.

Might check with @Blackbaud-TerryHelems but I believe this is not necessary here, as our config service does that for you.

Comment (Blackbaud-AlexKingman) [reply src/app/components/grid/index.html]: For example, a consumer would declare a key like so:
```
[settingsKey]="'event-list'"
```
NOT like this:
```
[settingsKey]="'skyux-spa-events-event-list'"
```

Comment (blackbaud-johnly) [reply src/app/components/grid/index.html]: I confirmed with Terry and removed the second sentence.

File: src/app/components/tile/index.html [hunk 2]
```diff
@@ -65,7 +65,7 @@
...
     >
-    Specifies a unique key for the UI Config Service to retrieve stored settings from a database. You must prefix the key with the name of your SPA. The UI Config Service applies sticky settings to tile dashboards and saves configuration settings for users. This property accepts <sky-code>string</sky-code> values. This property is for internal Blackbaud use only. For Blackbaud developers, the UI Config Service website describes how to <a href="https://docs.blackbaud.com/ui-config-service-docs/learn/tutorials/save-tile-configuration">apply sticky settings to tile dashboards</a>.
+    Specifies a unique key for the UI Config Service to retrieve stored settings from a database. You must prefix the key with the name of your SPA. The UI Config Service saves configuration settings for users to preserve the layout and collapsed state of tile dashboards. This property accepts <sky-code>string</sky-code> values. This property is for internal Blackbaud use only. For Blackbaud developers, the UI Config Service website describes how to <a href="https://docs.blackbaud.com/ui-config-service-docs/learn/integrations/skyux-components/tiles">apply sticky settings to tile dashboards</a>.
```
Comment (Blackbaud-JackMcElhinney) [inline src/app/components/tile/index.html]: Same removal is necessary for the tile as well I believe.

Comment (blackbaud-johnly) [reply src/app/components/tile/index.html]: Done. Thanks for catching that. I'm going back and forth between 2 different PRs and missed it.

**Pull request merged**